dashboard/src/pages/oauth-client/oauth-client.tsx

```diff
@@ -119,11 +120,16 @@ const OauthClient = () => {
     };
 
     const deleteClient = () => {
-        if (prompt('確認刪除 oAuthClient? 請輸入 delete') !== 'delete') {
-            alert('輸入錯誤');
-            return;
-        }
-        deleteApi();
+        dispatch(
+            dialogActions.open({
+                message: '請輸入 DELETE 完成刪除',
```

Comment:
這邊 messge 感覺可以跟 device 一樣，提醒刪除後就不能復原
```suggestion
              message: `刪除後將不可復原，請輸入 DELETE 完成刪除`
```